src/app/[locale]/(auth)/explore-story/page.tsx

```
@@ -54,14 +56,43 @@ export default function Index() {
+        });
+      }
+    }
+  }, [stories, page]);
+
   const storiesWithFavorites = useMemo(() => {
-    return stories && stories?.data && stories?.data.map((story: StoryType) => {
+    return items.map((story: StoryType) => {
       const isFavorite
         = favoriteStories
-          && favoriteStories?.some((favorite: any) => favorite.storyId === story.id);
+        && favoriteStories?.some((favorite: any) => favorite.storyId === story.id);
```

ESLint

[error] Expected indentation of 10 spaces

(style/indent-binary-ops)